zboxcore/sdk/common.go

```diff
@@ -121,3 +128,96 @@ func ValidateRemoteFileName(remotePath string) error {
 
 	return nil
 }
+
+func GetRoundFromSharders(sharders []string) (int64, error) {
```

Comment:
I think we need to get this from a single sharder, since it will be popular request and we will spam the network with it. we can keep some sort of round cache locally and refresh it periodically using pace (optional)